beacon-chain/rpc/beaconv1/validator_test.go

```diff
@@ -650,9 +664,10 @@ func Test_validatorStatus(t *testing.T) {
 		epoch     types.Epoch
 	}
 	tests := []struct {
-		name string
-		args args
-		want ethpb.ValidatorStatus
+		name    string
+		args    args
+		want    ethpb.ValidatorStatus
+		wantErr bool
```

Comment:
`wantErr` is not needed should you decide to remove the nil validator check. We can just have `require.NoError` everywhere.